packages/docs/docs/components/spline.mdx

```diff
@@ -0,0 +1,493 @@
+
+```tsx editor
+export default makeScene2D(function* (view) {
+  view.add(
+    <Spline lineWidth={6} stroke={'lightseagreen'}>
+      <Knot position={[-300, 0]} />
+      <Knot position={[-150, -100]} endHandle={[-100, 0]} />
+      <Knot position={[150, 100]} startHandle={[100, 0]} />
+      <Knot position={[300, 0]} />
+    </Spline>,
+  );
+});
+```
+
+Note that handle positions are relative to the knot's position.
+
+:::note
+
+Similar to using the `points` property, if no explicit handles are provided for
+a knot, the handles get calculated automatically so that the curve smoothly
+passes through the knot.
+
+:::
+
+#### Mirrored handles
+
+Handles are mirrored by default. This means that when we provide only one of the
+handles of a knot, the other one will implicitly be set to a flipped version of
+the provided handle.
+
+```tsx
+<Knot startHandle={[100, 50]} />
+// Is the same as
+<Knot startHandle={[100, 50]} endHandle={[-100, -50]} />
+```
+
+#### Broken knots
+
+Providing both `startHandle` and `endHandle` results in a so-called **broken
+knot**. Broken knots are very useful because they allow us to add sharp corners
+to our spline.
+
+```tsx editor
+export default makeScene2D(function* (view) {
+  view.add(
+    <Spline lineWidth={16} stroke={'lightseagreen'} closed>
+      <Knot position={[-50, -80]} startHandle={[0, 20]} endHandle={[90, 0]} />
+      <Knot position={[50, 0]} />
+      <Knot position={[-50, 80]} startHandle={[90, 0]} endHandle={[0, -20]} />
+    </Spline>,
+  );
+});
+```
+
+#### Blending between user handles and calculated handles
+
+By default, the auto-calculated handles get ignored when at least one of the
+`startHandle` or `endHandle` properties is provided. However, it is possible to
+blend between user-provided and auto-calculated handles by using the `auto`
+property.
+
+```tsx {6,13}
+<Spline lineWidth={16} stroke={'lightseagreen'} closed>
+  <Knot
+    position={[-50, -80]}
+    startHandle={[0, 20]}
+    endHandle={[90, 0]}
+    auto={0.5}
+  />
+  <Knot position={[50, 0]} />
+  <Knot
+    position={[-50, 80]}
+    startHandle={[90, 0]}
+    endHandle={[0, -20]}
+    auto={0.5}
+  />
+</Spline>
+```
+
+`auto` should be a value between `0` and `1` and represents the percentage of
+how much to blend between the user-provided handles and auto-calculated handles.
+A value of `1` means to use the auto handles while a value of `0` means using
+the user handles.
+
+:::tip
+
+`auto` is a compound signal, which means you can specify `startHandleAuto` and
+`endHandleAuto` to individually control the blend of each handle.
+
+```tsx {5-6}
+<Knot
+  position={[0, 0]}
+  startHandle={[-50, -50]}
+  endHandle={[30, 0]}
+  startHandleAuto={0.3}
+  endHandleAuto={0.8}
+/>
+```
+
+:::
+
+Since `auto` is a signal, it can also be animated.
+
+```tsx editor
+export default makeScene2D(function* (view) {
+  const knots: Knot[] = [];
+
+  view.add(
+    <Spline lineWidth={16} stroke={'lightseagreen'} closed>
```

Comment:
Should also have `lineJoin={'round'}` to prevent the quick transition from a broken handle to auto